### PR TITLE
Add app.vup.VupChat

### PR DESCRIPTION
**Pull request by lukehmcc**
### Please confirm your submission meets all the criteria

- [x] Please describe the application briefly.
Vup Chat is a cross platform messaging client for [bsky](https://bsky.app/) enhanced with more features.

- [x] The domain used for the application ID is [controlled by the application developers][appid-domain] and the [application id guidelines][appid] are followed.
- [x] I have read the and followed all the [Submission and licence requirements][reqs].
- [x] I have [built][build] and tested the submission locally.
- [x] I am an author/developer/upstream contributor of the project. If not, I contacted upstream developers about this submission. **Link:** (I am the author)

[appid-domain]: https://docs.flathub.org/docs/for-app-authors/requirements#control-over-domain
[appid]: https://docs.flathub.org/docs/for-app-authors/requirements#application-id
[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission

### CODE REVIEW

**Comment by bbhtt** [comment]
bot, build app.vup.VupChat

File: app.vup.Chat.yaml
Comment (bbhtt) [inline app.vup.Chat.yaml:1]: The filename needs to be renamed.

Comment (lukehmcc) [reply app.vup.Chat.yaml:1]: right right sorry

**Comment by bbhtt** [comment]
bot, build app.vup.VupChat

**Comment by bbhtt** [comment]
How does one get the login/password? There is no sign up button.

Also can you test the above build?

**Comment by lukehmcc** [comment]
Oh, visual bug is hiding the indicator on how to sign up.
here's the link
https://blueskyfeeds.com/en/faq-app-password

I tested it and it works for me.

**Pull request closed without merging**